scalahost-sbt/src/sbt-test/sbt-scalahost/semantic-example/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("org.scalameta" % "sbt-scalahost" % sys.props("plugin.version"))
```

Comment:
See https://github.com/scalameta/scalameta/blob/6ce2620fb37c0ce1b1cae81d547c922be569f9ba/build.sbt#L245

Scripted starts `sbt` in a temporary directory.